edc-controlplane/edc-controlplane-postgresql-hashicorp-vault-legacy/README.md

```diff
@@ -0,0 +1,178 @@
+# EDC Control-Plane backed by [Postgresql](https://www.postgresql.org/) and [HashiCorp vault](https://www.vaultproject.io/docs)
+
+## Building
+
+```shell
+./gardlew :edc-controlplane:edc-controlplane-postgresql-hashicorp-vault:dockerize
+```
+
+## Configuration
+
+Listed below are configuration keys needed to get the `edc-controlplane-postgresql-hashicorp-vault` up and running.
+Details regarding each configuration property can be found at the [documentary section of the EDC](https://github.com/eclipse-edc/Connector/tree/main/docs).
+
+| Key                                              | Required | Example                                                                      | Description                |
+|--------------------------------------------------|----------|------------------------------------------------------------------------------|----------------------------|
+| edc.api.auth.key                                 |          | password                                                                     | default value: random UUID |
+| web.http.default.port                            | X        | 8080                                                                         |                            |
+| web.http.default.path                            | X        | /api                                                                         |                            |
+| web.http.data.port                               | X        | 8181                                                                         |                            |
+| web.http.data.path                               | X        | /data                                                                        |                            |
+| web.http.validation.port                         | X        | 8182                                                                         |                            |
+| web.http.validation.path                         | X        | /validation                                                                  |                            |
+| web.http.control.port                            | X        | 9999                                                                         |                            |
+| web.http.control.path                            | X        | /api/controlplane/control                                                    |                            |
+| web.http.ids.port                                | X        | 8282                                                                         |                            |
```

Comment:
Are these `ids.` properties used anymore?

Reply:
i don't think they are, but the README's are regenerated when we publish the helm chart. it's possible that the regenerated version never found it's way back into `main`

Reply:
also, this is only in the `legacy` chart, which will go away eventually